src/charm.py

```diff
@@ -154,11 +154,14 @@ class GrafanaAgentMachineCharm(GrafanaAgentCharm):
     mandatory_relation_pairs = {
         "cos-agent": [  # must be paired with:
             {"grafana-cloud-config"},  # or
-            {"send-remote-write", "logging-consumer", "grafana-dashboards-provider"},
+            {"send-remote-write"},  # or
+            {"logging-consumer"},  # or
+            {"grafana-dashboards-provider"},
         ],
         "juju-info": [  # must be paired with:
             {"grafana-cloud-config"},  # or
-            {"send-remote-write", "logging-consumer"},
+            {"send-remote-write"},  # or
+            {"logging-consumer"},
```

Comment:
Are we sure this refactor is ok?

`juju-info` incoming relation needs an outgoing `grafana-cloud-config` or these two: `send-remote-write`, `logging-consumer` in order to set the `ActiveStatus`

Reply:
I was surprised too but turns out it's desired behavior: https://github.com/canonical/grafana-agent-k8s-operator/pull/205

Reply:
Oh!! I did bot remember that!!! 🤦🏼‍♂️